code/game/objects/items/devices/scanners.dm

```diff
@@ -264,147 +271,136 @@ GENE SCANNER
 							<td><font color='#00cc66'><b>[CEILING(tox_loss,1)]</b></font></td>\
 							<td><font color='#33ccff'><b>[CEILING(oxy_loss,1)]</b></font></td></tr>"
 
-			for(var/o in damaged)
-				var/obj/item/bodypart/org = o //head, left arm, right arm, etc.
-				dmgreport += "<tr><td><font color='#cc3333'>[capitalize(org.name)]:</font></td>\
-								<td><font color='#cc3333'>[(org.brute_dam > 0) ? "[CEILING(org.brute_dam,1)]" : "0"]</font></td>\
-								<td><font color='#ff9933'>[(org.burn_dam > 0) ? "[CEILING(org.burn_dam,1)]" : "0"]</font></td></tr>"
+			if(mode == SCANNER_VERBOSE)
+				for(var/obj/item/bodypart/limb as anything in damaged)
+					dmgreport += "<tr><td><font color='#cc3333'>[capitalize(limb.name)]:</font></td>\
+									<td><font color='#cc3333'>[(limb.brute_dam > 0) ? "[CEILING(limb.brute_dam,1)]" : "0"]</font></td>\
```

Comment:
Don't use tab alignment, this is banned in the style guide.
Split this into three `dmgreport += ""` instead.